example/poll.c

```diff
@@ -61,6 +65,15 @@ static int fsel_path_index(const char *path)
 	return ch <= '9' ? ch - '0' : ch - 'A' + 10;
 }
 
+static void fsel_destroy(void *private_data)
+{
+	(void)private_data;
+
+	fsel_stop = true;
```

Comment:
... and if this is an atomic then I think you need to access it via the appropriate atomic_load function.

In practice I don't think this makes a difference on modern CPUs (and less so for a bool) but you're using the atomic api in one place and not in others and that's technically undefined behaviour.

Reply:
I might be wrong, but AFAIK it is sufficient to specify _Atomic for the type for C11 atomic operations 

https://developers.redhat.com/blog/2016/01/14/toward-a-better-use-of-c11-atomics-part-1#solutions_adopted_in_c11

See "Intuitive Syntax For Operations On Atomic Types"

Reply:
I may have been working from an older version of the spec.  This is great if true.